File: app/src/main/java/com/wb/logistics/ui/delivery/DeliveryDataBuilderImpl.java
package com.wb.logistics.ui.delivery;

import androidx.annotation.NonNull;

import com.wb.logistics.mvp.model.base.BaseItem;
import com.wb.logistics.ui.delivery.res.DeliveryResourceProvider;

import java.util.List;

public class DeliveryDataBuilderImpl implements DeliveryDataBuilder {

    @NonNull
    private final DeliveryResourceProvider resourceProvider;

    public DeliveryDataBuilderImpl(@NonNull DeliveryResourceProvider resourceProvider) {
        this.resourceProvider = resourceProvider;
    }

    @NonNull
    @Override
    public List<BaseItem> buildFlights() {
        return null;
    }

}

File: app/src/main/java/com/wb/logistics/ui/delivery/DeliveryDataBuilderImpl.kt
package com.wb.logistics.ui.delivery

import com.wb.logistics.mvvm.model.base.BaseItem
import com.wb.logistics.ui.delivery.res.DeliveryResourceProvider

class DeliveryDataBuilderImpl(private val resourceProvider: DeliveryResourceProvider) :
    DeliveryDataBuilder {
    override fun buildFlights(): List<BaseItem> {
        return ArrayList()
    }
}